ERCS/erc-transfer-with-authorization.md

```diff
@@ -0,0 +1,536 @@
+---
+eip: 3009
```

Comment:
```suggestion
eip: 7758
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

Reply:
@abcoathup Thank you for the review
https://github.com/ethereum/ERCs/pull/598/commits/ea63268cda2ca0a21eb7d7eec173296fc9f2ed66
* Update eip number to 7758
* Change filename